KrishnaSharma206.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@KrishnaSharma206"
+project: "An AI powered Robot"
+---
+
+# Object Recognition Warehouse Robot
+
+## Summary
+
+A robot that recohnizes certain shapes and navigates it's way accordingly, I am going to build this robot because I liked the idea of the amazon warehouse robot and the projects made by a guy named James Bruton, and I really wanna learn about object recognition through this project. I already have some experience making robots using arduino, so it will be something new for me.
+
+## Plan
+
+I will be using the jetson nano as the brain of this project, the raspberry pi wont be enough for this and the jetson nano will be a better alternative
+, I will be using a camera for the detection and I am planning to add more sensors and actuators to it in the future.
+
+The Steps:
+1. Learn how to use the jetson nano
+2. Connect my laptop to the jetson nano using a vnc
+3. Train the AI model on the jetson nano using basic shapes for easier recognition
+4. Build the robot, then program the robot using python
+5. Test the robot.
+
+## Budget
+
+| Product                | Supplier/Link                                               |  Cost   |
+| ---------------        | ----------------------------------------------------------- | ------- |
+| Nvidia Jetson nano     | https://www.arrow.com/en/products/945-13450-0000-100/nvidia | $195.00 |
```

Comment:
<img width="1007" alt="Screenshot 2022-12-30 at 6 46 52 PM" src="https://user-images.githubusercontent.com/72365100/210118527-0e667ec4-f5f0-4c5a-b9b3-00148f83ecdb.png">

This appears to be out of stock; do you have alternatives?

Reply:
I could use the beaglebone ai, but it would be a bit hard to learn

Reply:
I found a website, which has the board in stock, I mentioned it in the file